.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,28 @@
+name: Go
+on:
+  - push
+  - pull_request
+jobs:
+
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Set up Go 1.13
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v1
+
+    - name: Test
+      run: go test -v ./...
+
+    - name: Build
+      run: go build cmd/proxy/*.go
+      env:
+        GOPROXY: https://proxy.golang.org
+        GO111MODULE: "on"
```

Comment:
@ClaytonNorthey92 can you add a newline (with no indent) at the end of this file?

Reply:
just did 👍